### PR TITLE
Complete account api ready for code review

### PR DESCRIPTION
**Pull request by kenendo12**

### CODE REVIEW

File: aspnet/RVTR.Account.WebApi/Controllers/PaymentController.cs [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    public async Task<IActionResult> Delete(int id)
+    {
+      try
+      {
+        await _unitOfWork.PaymentRepository.Delete(id);
+        await _unitOfWork.Complete();
+
+        return Ok();
+      }
+      catch
```
Comment (markmoorerev) [inline aspnet/RVTR.Account.WebApi/Controllers/PaymentController.cs:47]: which exception are you catching?

File: aspnet/RVTR.Account.WebApi/Controllers/ProfileController.cs [hunk 2]
```diff
@@ -17,92 +18,90 @@ namespace RVTR.Account.WebApi.Controllers
...
       try
       {
-        await _unitOfWork.Profile.DeleteAsync(id);
-        await _unitOfWork.CommitAsync();
+        await _unitOfWork.ProfileRepository.Delete(id);
+        await _unitOfWork.Complete();
 
         return Ok();
       }
       catch
```
Comment (markmoorerev) [inline aspnet/RVTR.Account.WebApi/Controllers/ProfileController.cs:45]: which exception are you catching?

File: aspnet/RVTR.Account.WebApi/Program.cs [hunk 3]
```diff
@@ -9,10 +10,12 @@ namespace RVTR.Account.WebApi
   /// <summary>
   ///
```
Comment (markmoorerev) [inline aspnet/RVTR.Account.WebApi/Program.cs:11]: description?

**Comment by fredbelotte** [comment]
the work in this PR has been broken into several new PRs as part of the change in the project scope.

**Pull request closed without merging**